codechecker_lib/arg_handler.py

```diff
@@ -90,30 +84,21 @@ def handle_server(args):
     """
     Starts the report viewer server.
     """
-    if not host_check.check_zlib():
-        LOG.error("zlib error")
-        sys.exit(1)
+    host_check.check_zlib()
```

Comment:
The check_zlib method will terminate the execution if zlib is not found. Having additional termination here is redundant.